cmd/explaintest/r/explain_easy.result

```diff
@@ -172,26 +172,31 @@ Union	26000.00	root
 ├─HashAgg	16000.00	root		group by:Column#10, funcs:firstrow(Column#12)->Column#10
 │ └─Union	16000.00	root		
 │   ├─StreamAgg	8000.00	root		group by:test.t2.c1, funcs:firstrow(test.t2.c1)->Column#12, funcs:firstrow(test.t2.c1)->Column#10
-│   │ └─IndexReader	10000.00	root		index:IndexFullScan
-│   │   └─IndexFullScan	10000.00	cop[tikv]	table:t2, index:c1(c1)	keep order:true, stats:pseudo
+│   │ └─IndexReader	8000.00	root		index:StreamAgg
+│   │   └─StreamAgg	8000.00	cop[tikv]		group by:test.t2.c1, 
+│   │     └─IndexFullScan	10000.00	cop[tikv]	table:t2, index:c1(c1)	keep order:true, stats:pseudo
 │   └─StreamAgg	8000.00	root		group by:test.t2.c1, funcs:firstrow(test.t2.c1)->Column#12, funcs:firstrow(test.t2.c1)->Column#10
-│     └─IndexReader	10000.00	root		index:IndexFullScan
-│       └─IndexFullScan	10000.00	cop[tikv]	table:t2, index:c1(c1)	keep order:true, stats:pseudo
+│     └─IndexReader	8000.00	root		index:StreamAgg
+│       └─StreamAgg	8000.00	cop[tikv]		group by:test.t2.c1, 
```

Comment:
The plan changed since the original way of calculating the cost is wrong?
```
partialAgg.SetCost(cop.cost())
```
Is there?

Reply:
Yes, if the agg can be pushed down, when accumulating final-agg's(running on TiDB) cost, we use `t.addCost(p.GetCost(inputRows, true, false))`, but actually the pointer `p` here is `partial-agg` instead of `final-agg`, so we get a wrong cost for plans which have a pushed-down agg.